google-maps/src/main/java/org/odk/collect/googlemaps/GoogleMapFragment.java

```diff
@@ -599,7 +599,7 @@ private void updateLocationIndicator(LatLng loc, double radius) {
             );
         }
         if (accuracyCircle == null) {
-            int stroke = ContextUtils.getThemeAttributeValue(requireContext(), com.google.android.material.R.attr.colorPrimaryDark);
+            int stroke = ContextUtils.getThemeAttributeValue(requireContext(), androidx.appcompat.R.attr.colorPrimaryDark);
```

Comment:
Again, we should take the opportunity here to replace this with a Material Color